backend/src/graphql/resolver/UserResolver.ts

```diff
@@ -293,16 +291,14 @@ export class UserResolver {
         logger.info('redeemCode found contributionLink', contributionLink)
         if (contributionLink) {
           dbUser.contributionLinkId = contributionLink.id
-          // TODO this is so wrong
```

Comment:
@clauspeterhuebner here the todo was fixed

Reply:
ok...